src/coprocessor/readpool_impl.rs

```diff
@@ -13,6 +14,7 @@ pub fn build_read_pool<E: Engine, R: FlowStatsReporter>(
     config: &CoprReadPoolConfig,
     reporter: R,
     engine: E,
+    sender: mpsc::Sender<SplitHub>,
```

Comment:
Why need this `sender`? we already have a report.

Reply:
Good idea. I will try to merge them.